test/general-tests/hardhat-run/check.ts

```diff
@@ -0,0 +1,8 @@
+// import { hardhatStarknetCompile, hardhatStarknetRun } from "../../utils/cli-functions";
+// import { ensureEnvVar } from "../../utils/utils";
+
+console.log("$0: Testing hh run with --starknet-network is currently disabled");
```

Comment:
Vars with a dollar sign make no sense in js tests. We can simply omit printing the script name ($0) wherever we had it.

Reply:
https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/251/commits/f053778ce3a76723ae6aa82cf34fc9b8a9623dc4